### DISCUSSION TITLE
Is it possible to determine which docker container has the most HD access?

### DISCUSSION THREAD

**Discussion by RKLBusinessDevelopment** [henrygd/beszel#108] [Support] [answered]
Thanks for creating Beszel, it's a very useful tool. Is there a possibility to see which docker container has most HD accesses? Background is that I would like to know which containers cause constant HD (write) access on my NAS, so I can either disable or pause these containers to get rid of the annoying HD noise they cause. I am not sure if that is possible with Beszel, I checked the UI but didn't find a solution yet. Any pointers are welcome, thank you.

**Comment by henrygd** [comment, marked as answer]
Block I/O for containers is not implemented yet, but probably will be at some point.

**Reply by RKLBusinessDevelopment** [reply]
Thank you, I would love such feature. Thanks for your reply (and Beszel), much appreciated!
